### PR TITLE
Updating localization-de.po

### PR DESCRIPTION
**Pull request by sfahrenholz**
# Fix some misspelling
# create new translations for not translated strings

I updating the german translation. I hope this is the correct file..

### CODE REVIEW

File: Localizations/localization-de.po [hunk 1]
```diff
@@ -128,7 +128,7 @@ msgstr "Schwaches Passwort"
 
 #: Server/webroot/ngax/scripts/controllers/EditBackupController.js:292
 msgid "Your passphrase is easy to guess. Consider changing passphrase."
-msgstr "Dein Passwort is leicht zu erraten. Nimm lieber etwas Komplizierteres."
+msgstr "Dein Passwort is leicht zu erraten. Nimm lieber etwas komplizierteres."
```
Comment (renestach) [inline Localizations/localization-de.po]: ist

File: Localizations/localization-de.po [hunk 2]
```diff
@@ -1328,7 +1328,7 @@ msgstr "Pause"
 
 #: Server/webroot/ngax/templates/pause.html:5
 msgid "Until resumed"
-msgstr "Until resumed"
+msgstr "Bis zur wiederaufnahme"
```
Comment (renestach) [inline Localizations/localization-de.po]: Wiederaufnahme

File: Localizations/localization-de.po (hunk 1 above)
Comment (renestach) [inline Localizations/localization-de.po]: Komplizierteres groß schreiben?

File: Localizations/Server/de.po [hunk 3]
```diff
@@ -47,56 +47,56 @@ msgstr "Über {0}"
...
 #: ../../Duplicati/Server/webroot/ngax/templates/settings.html:5
 msgid "Access to user interface"
-msgstr ""
+msgstr "Zugriff zur Benutzeroberfläche"
 
 #: ../../Duplicati/Server/webroot/ngax/scripts/services/EditUriBuiltins.js:600
 #: ../../Duplicati/Server/webroot/ngax/templates/backends/azure.html:7
 msgid "Account name"
-msgstr ""
+msgstr "Kontoname"
```
Comment (renestach) [inline Localizations/Server/de.po]: Wollen wir solche Wörter trennen? Konto-Name?

**Comment by agrajaghh** [comment]
@sfahrenholz Thanks a lot for your translation. But we are managing the translation process at transifex:
https://www.transifex.com/duplicati/duplicati/dashboard/

We would be happy if you can help us there (you can use your github account to login) 😊

**Pull request closed without merging**